apps/web-mzima-client/src/app/shared/components/onboarding/onboarding.component.html

```diff
@@ -57,11 +57,11 @@ <h2>{{ step.title }}</h2>
 
 <ng-template #finalStepContent>
   <p>
-    How exciting! Now you know the most important features and you’re ready to start using Ushahidi
+    How exciting! Now that you know the most important features and you’re ready to start using Ushahidi
     platform.
```

Comment:
Could you just add a little comma in above sentence: "Now that you know the most important features, you're ready to start using Ushahidi."